packages/shared/src/svg/LogoIcon.tsx

```diff
@@ -20,10 +20,10 @@ export default function LogoIcon({ className }: LogoIconProps): ReactElement {
         className={className?.group}
       >
         <path
-          d="M26.633 8.69l-3.424-3.431 1.711-3.43 5.563 5.575c.709.71.709 1.861 0 2.572l-6.847 6.86c-.709.711-1.858.711-2.567 0a1.821 1.821 0 010-2.571l5.564-5.575z"
+          d="M29.5925 9.99823L25.7884 6.1862L27.6895 2.37549L33.8703 8.5693C34.6579 9.35848 34.6579 10.638 33.8703 11.4272L26.2629 19.0506C25.4753 19.8398 24.1985 19.8398 23.411 19.0506C22.6234 18.2614 22.6234 16.9819 23.411 16.1927L29.5925 9.99823Z"
```

Comment:
What are the changes here?

Reply:
No changes really, mostly viewport of svg changed, so the paths changed too.
I copied the SVG from figma

Reply:
![SCR-20241113-lwrn](https://github.com/user-attachments/assets/35dac94f-c868-4aa9-a3c2-dca02487bb3e)

This shows the difference